h2o-py/tests/testdir_munging/unop/pyunit_vec_math_ops.py

```diff
@@ -14,14 +14,24 @@ def vec_math_ops():
     seed0 = random.random()
     random.seed(seed0)
     print("Using seed %r" % seed0)
-
+    badDataTan = [-math.pi/2, math.pi/2, -math.pi*1.5, math.pi*1.5, -math.pi*2.5, math.pi*2.5]  # tan value can blow up near this values
     sin_cos_tan_atan_sinh_cosh_tanh_asinh_data = [[random.uniform(-10,10) for r in range(10)] for c in range(10)]
+    # check to make sure random data do not have arguments near bad values for tan.
```

Comment:
Michal:

Because if you look at the slope of tan(x) as x goes towards +/-pi/2, +/-1.5pi, +/-2.5pi, the slope is very steep.  As a result, there is more divergence between the Java tan() and np tan() function exceeding 1e-12.  It is about 1e-8.  That is why I am filtering out those values in order for the test to pass.  Since we do not implement the tan functions ourselves, there is nothing we can do about the divergence.